src/components/ItemBook.css

```diff
@@ -45,10 +83,46 @@
 
 .progress-text .stat-text {
   font-size: 1.4rem;
+  color: grey;
+}
+
+.progress-info-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
 }
 
 .progress-info {
   display: flex;
   flex-direction: column;
-  gap: 1rem;
+  gap: 0.4rem;
+}
+
+.progress-info .chapter-title {
+  font-family: 'Roboto Slab', serif;
+  color: grey;
+}
+
+.progress-info .chapter-number {
+  font-family: 'Roboto Slab', serif;
+  font-size: 1.4rem;
+}
```

Comment:
- [ ] It would be good if the styling is the same as the one provided on Zeplin to meet the requirements. Kindly check the differences shown below and modify them 👇
- Your design
<img width="960" alt="Bookstore - andres-condezo github io" src="https://user-images.githubusercontent.com/46851023/155856706-032ff24b-f6c8-4384-b577-8135f5b35d21.png">

- Zeplin design 
![Bookstore CMS - Microverse Portfolio Projects - Zeplin - app zeplin io](https://user-images.githubusercontent.com/46851023/155856855-b8d90ca8-06d2-43d7-bdc4-c24b5f99074d.png)
Some of the differences are marked with black colour, kindly compare to see other differences. Use similar spacing and add the missing elements.

Reply:
Hi @ZeenatLawal, thank you for the feedback, I will correct the styles right now.